### PR TITLE
Added Dozzle

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Adding Dozzle.

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [ ] Use github checklists. When solved, check the box and explain the answer.

**Requirements**
Check all boxes as they are completed

- [ ] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [ ] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/dozzle/dozzle.aarch64.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+services:
+  dozzle:
+    image: amir20/dozzle:latest
```
Comment (nemchik) [inline compose/.apps/dozzle/dozzle.aarch64.yml:3]: We can leave off `:latest`
```suggestion
    image: amir20/dozzle
```

File: compose/.apps/dozzle/dozzle.armv7l.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+services:
+  dozzle:
+    image: amir20/dozzle:latest
```
Comment (nemchik) [inline compose/.apps/dozzle/dozzle.armv7l.yml:3]:
```suggestion
    image: amir20/dozzle
```

File: compose/.apps/dozzle/dozzle.labels.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      com.dockstarter.appinfo.description: "Dozzle is a simple, lightweight application that provides you with a web based interface to monitor your Docker container logs live."
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.nicename: "dozzle"
+      com.dockstarter.appvars.dozzle_enabled: "false"
+      com.dockstarter.appvars.dozzle_network_mode: ""
+      com.dockstarter.appvars.dozzle_port_8080: "8080"
+      com.dockstarter.appvars.dozzle_base: "/"
+      com.dockstarter.appvars.dozzle_level: "info"
+      com.dockstarter.appvars.dozzle_tailsize: "300"
+      com.dockstarter.appvars.dozzle_filter: ""
```
Comment (nemchik) [inline compose/.apps/dozzle/dozzle.labels.yml:13]: Sort alphabetically
```suggestion
      com.dockstarter.appvars.dozzle_base: "/"
      com.dockstarter.appvars.dozzle_enabled: "false"
      com.dockstarter.appvars.dozzle_filter: ""
      com.dockstarter.appvars.dozzle_level: "info"
      com.dockstarter.appvars.dozzle_network_mode: ""
      com.dockstarter.appvars.dozzle_port_8080: "8080"
      com.dockstarter.appvars.dozzle_tailsize: "300"
```

File: compose/.apps/dozzle/dozzle.x86_64.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+services:
+  dozzle:
+    image: amir20/dozzle:latest
```
Comment (nemchik) [inline compose/.apps/dozzle/dozzle.x86_64.yml:3]:
```suggestion
    image: amir20/dozzle
```

File: compose/.apps/dozzle/dozzle.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  dozzle:
+    container_name: dozzle
+    environment:
+      - PGID=${PGID}
+      - PUID=${PUID}
+      - TZ=${TZ}
+      - DOZZLE_BASE${DOZZLE_BASE}
+      - DOZZLE_LEVEL=${DOZZLE_LEVEL}
+      - DOZZLE_TAILSIZE=${DOZZLE_TAILSIZE}
+      - DOZZLE_FILTER=${DOZZLE_FILTER}
```
Comment (nemchik) [inline compose/.apps/dozzle/dozzle.yml:11]:
```suggestion
      - DOZZLE_BASE${DOZZLE_BASE}
      - DOZZLE_FILTER=${DOZZLE_FILTER}
      - DOZZLE_LEVEL=${DOZZLE_LEVEL}
      - DOZZLE_TAILSIZE=${DOZZLE_TAILSIZE}
      - PGID=${PGID}
      - PUID=${PUID}
      - TZ=${TZ}
```

Comment (angrycuban13) [reply compose/.apps/dozzle/dozzle.yml]: 81b0cf8

File: compose/.apps/dozzle/dozzle.x86_64.yml (hunk 4 above)
Comment (angrycuban13) [reply compose/.apps/dozzle/dozzle.x86_64.yml]: 81b0cf8

File: compose/.apps/dozzle/dozzle.labels.yml (hunk 3 above)
Comment (angrycuban13) [reply compose/.apps/dozzle/dozzle.labels.yml]: 81b0cf8

File: compose/.apps/dozzle/dozzle.armv7l.yml (hunk 2 above)
Comment (angrycuban13) [reply compose/.apps/dozzle/dozzle.armv7l.yml]: 81b0cf8

File: compose/.apps/dozzle/dozzle.aarch64.yml (hunk 1 above)
Comment (angrycuban13) [reply compose/.apps/dozzle/dozzle.aarch64.yml]: 81b0cf8

File: compose/.apps/dozzle/dozzle.yml (hunk 5 above)
Comment (angrycuban13) [reply compose/.apps/dozzle/dozzle.yml]: 81b0cf8

**Pull request merged**